lib/ret/api/token_module.ex

```diff
@@ -0,0 +1,111 @@
+defmodule Ret.Api.TokenModule do
+  @moduledoc """
+  This module should not be used directly.
+
+  It is intended to be used by Guardian.
+  """
+  alias Ret.{Account, Repo}
+  alias Ret.Api.Credentials
+  @behaviour Guardian.Token
+
+  @doc """
+  No concept of validating signature so we just decode the token
+  """
```

Comment:
Forgot to mention. Does this imply that we are forgoing the benefits of a signed JWT token? Does that impact security? Perhaps this aspect is worth a deeper review.

Reply:
I'm not 100% sure about this, but I don't think our API tokens need to be signed because 
- there are no third parties that issue tokens that we'd need to validate (it's either in our database or it isn't) and
- there is no data that needs to be trusted in the token itself.

As an example, in Auth0, signatures are required in the following flow:
- A user tries to sign in to the application.
- Auth0 creates a signed token for the user.
- The application validates the token signature against the Auth0 public key to make sure the token came from Auth0.
- The application can trust the claims in the token if there are any.

Reply:
I think we should hop on a call to discuss this in more detail. I'm probably just forgetting conversations we've had in the past where we reasoned through using JWTs.

Reply:
Had that call. The main concern I had was incorrect, because we aren't actually using JWTs in the auth header, and that's why we aren't doing any verification. We use a raw token in the header and use this custom token_module to convert that to a `Credentials` object.